grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -1239,7 +1239,7 @@ class OrderService {
             // Drop mv temp table if somehow it still exists
             "DROP TABLE IF EXISTS order_summary_mv_temp;",
             // Create temp mv table from sql view (to shorten the time when MV is unavailable)
-            "CREATE TABLE order_summary_mv_temp AS SELECT * FROM order_summary;",
+            "CREATE TABLE order_summary_mv_temp AS SELECT DISTINCT * FROM order_summary;",
```

Comment:
This feels like an antipattern but it's probably the easiest way to solve the problem for now. We should also be looking for the ways in which we might encounter duplicate records and/or add a unique constraint across all columns.

But since that would likely cause downtime requiring manual intervention this feels like an ok approach for now.

Reply:
Sorry for the duplicate comments (since deleted). GitHub was not saving the comment, so I had to retry a few times.